Projects/private-ca/roles/vault/tasks/main.yml

```diff
@@ -3,6 +3,10 @@
   wait_for:
     port: 8200
 
+- name: Wait for a bit.
+  pause:
+    seconds: 20
+
```

Comment:
Need to somehow better handle the fact that Vault is not ready for initialization immediately after the service is started.